omod/src/main/webapp/resources/htmlforms/sierra_leone/vitals.xml

```diff
@@ -549,9 +549,47 @@
             </fieldset>
 
         </includeIf>
+<fieldset>
```

Comment:
Could you please fix the indentation of this new section?

We don't have official style guidelines at PIH as far as I'm aware, but please give https://google.github.io/styleguide/htmlcssguide.html a look through for some solid general guidance.

Reply:
For "fixups" you can use `git commit --amend` to update the last commit. You'll then need to use `--force` with your `git push` command.

Reply:
I thought i just needed to amend my local file then git push fork mybranch as discussed yesterday